README.md

```diff
@@ -18,7 +18,7 @@ Edit the required configruation values in the .env file.
 
 ## Generate Validators Keystores
 ```sh
-docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:v0.14.0 create cluster --withdrawal-address="0x000000000000000000000000000000000000dead" --num-validators=1 --nodes=5 --threshold=3 --network=goerli
+docker run --rm -v "$(pwd):/opt/charon" ghcr.io/obolnetwork/charon:v0.14.3 create cluster --withdrawal-address="0x000000000000000000000000000000000000dead" --num-validators=1 --nodes=5 --threshold=3 --network=goerli
```

Comment:
can remove `--network=goerli` as goerli is the default network

Reply:
Let's leave it as that is not a very expected default! Also. When we finish audit we should make mainnet the default. Otherwise you risk a mainnet invalid deposit imo.